java/src/jmri/jmrix/pi/ConnectionConfig.java

```diff
@@ -94,10 +94,6 @@ public boolean getDisabled(){
     public void setDisabled(boolean disable){
        this.disabled=disable;
     }
-
-   @Override
-   public void dispose(){
```

Comment:
This is the only instance of dispose() in a ConnectionConfig that did not call super.

@pabender Did this intentionally not call super.dispose()?